stable/oidc-static/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Default values for oidc-static.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  nginx:
+    repository: nginx
+    tag: 1.14.2
+    pullPolicy: IfNotPresent
+  init:
+    repository: anchorfree/oidc-static
+    tag: 53f0ae9c4c55034ba42af345b5b2beb09906e431
+    pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  paths: []
+  hosts:
+    - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources:
+  init:
+    limits:
+      cpu: 16m
+      memory: 32Mi
+#    requests:
+#      cpu: 100m
+#      memory: 128Mi
+  nginx:
+    limits:
+      cpu: 16m
```

Comment:
are you sure that 16 milicors is enough?

Reply:
looks so, and here it is a minimal set of resources to start app, for more we can override it in values when necessary